src/price/currency-format.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { SFC } from 'react';
+
+export type CurrencyFormatProps = {
+  locale: string,
+  currency: string,
+  value: string
+}
+
+const CurrencyFormat: SFC<CurrencyFormatProps> = ({locale, currency, value }) => (
+  <div>
+      {(locale) ? new Intl.NumberFormat(locale, { style: 'currency', currency: currency, maximumFractionDigits: 10
```

Comment:
We can allow passing in an optional `maximumFractionDigits` prop that defaults to whatever we choose (it should probably be `null` by default so that it uses the correct significant digits for the currency in question) and then add it to the index string.

Reply:
@dfcowell I don't see we can use the same instance because for instance us-en will default to 2 fraction digits. That is not good for crypto right. We will need more digits. I guess for now it is okay to have one single instance with 10 fraction digits for instance. Would this be a case for using redux or should be a global NumberFormat instance on window so we can just reference it there? Cheers